lib/stimpack.rb

```diff
@@ -44,6 +41,8 @@ def [](name)
     config/initializers
     config/routes
   )
+  # (0) / -> (1) /packs -> (2) /packs/foo/bar
+  @config.max_levels = 2
```

Comment:
Does this imply that folks can have N-level nested packs?

Reply:
And do we really want to support that? It won't work with the rest of the ruby at scale ecosystem.

Reply:
I'll remove this config.